jobsdb/jobsdb.go

```diff
@@ -276,16 +283,17 @@ multiple users of JobsDB
 dsRetentionPeriod = A DS is not deleted if it has some activity
 in the retention time
 */
-func (jd *HandleT) Setup(clearAll bool, tablePrefix string, retentionPeriod time.Duration, toBackup bool) {
+func (jd *HandleT) Setup(clearAll bool, tablePrefix string, retentionPeriod time.Duration, backupSettings *BackupSettingsT) {
 
 	var err error
 	psqlInfo := GetConnectionString()
 	jd.assert(tablePrefix != "", "tablePrefix received is empty")
 	jd.tablePrefix = tablePrefix
 	jd.dsRetentionPeriod = retentionPeriod
-	jd.toBackup = toBackup
 	jd.dsEmptyResultCache = map[dataSetT]map[string]map[string]map[string]bool{}
 
+	jd.BackupSettings = backupSettings
```

Comment:
Initialize the backup setting here instead of main.go?

Reply:
moved to jobsdb, but not as jd interface implementation but as factory.